packages/libs/src/sdk/api/generated/default/apis/TracksApi.ts

```diff
@@ -68,6 +74,48 @@ export interface StreamTrackRequest {
  */
 export class TracksApi extends runtime.BaseAPI {
 
+    /**
+     * @hidden
+     * Download the original or MP3 file of a track
+     */
+    async downloadTrackRaw(params: DownloadTrackRequest, initOverrides?: RequestInit | runtime.InitOverrideFunction): Promise<runtime.ApiResponse<void>> {
+        if (params.trackId === null || params.trackId === undefined) {
+            throw new runtime.RequiredError('trackId','Required parameter params.trackId was null or undefined when calling downloadTrack.');
+        }
+
+        const queryParameters: any = {};
+
+        if (params.userSignature !== undefined) {
+            queryParameters['user_signature'] = params.userSignature;
+        }
+
+        if (params.userData !== undefined) {
+            queryParameters['user_data'] = params.userData;
+        }
+
+        if (params.original !== undefined) {
+            queryParameters['original'] = params.original;
+        }
+
+        const headerParameters: runtime.HTTPHeaders = {};
+
+        const response = await this.request({
+            path: `/tracks/{track_id}/download`.replace(`{${"track_id"}}`, encodeURIComponent(String(params.trackId))),
+            method: 'GET',
+            headers: headerParameters,
+            query: queryParameters,
+        }, initOverrides);
+
+        return new runtime.VoidApiResponse(response);
```

Comment:
Might have to make an override/mask for this fn as well, like we did for stream

Reply:
yup